Formula/lib/libpq.rb

```diff
@@ -1,6 +1,6 @@
 class Libpq < Formula
   desc "Postgres C API library"
-  homepage "https://www.postgresql.org/docs/15/libpq.html"
+  homepage "https://www.postgresql.org/docs/16/libpq.html"
```

Comment:
Thanks, @gasi. It makes me wonder if this makes more sense:

```suggestion
  homepage "https://www.postgresql.org/docs/current/libpq.html"
```

Reply:
Yeah, I think that's better

Reply:
:agree:, updated to use `current` instead.